pkg/cmd/evict_pod.go

```diff
@@ -64,19 +65,22 @@ func NewCmdModifySecret(streams genericclioptions.IOStreams) *cobra.Command {
 	}
 
 	cmd.Flags().BoolVar(&o.printVersion, "version", false, "prints version of plugin")
+	cmd.Flags().StringVar(&o.label, "label", "", "specify a label to evict pods with")
 	o.configFlags.AddFlags(cmd.Flags())
 
 	return cmd
 }
 
 // Complete sets all information required for updating the current context
 func (o *EvictPodOptions) Complete(cmd *cobra.Command, args []string) error {
-	o.args = args
 
-	if len(o.args) > 0 {
-		o.podName = o.args[0]
+	if len(args) == 0 {
```

Comment:
apologies for super delay in reviewing this PR. if only label is provided, the len(args) can be 0 right?